ios/Source/Core/LRCookieCallback.h

```diff
@@ -0,0 +1,18 @@
+//
+//  LRCookieCallback.h
+//  Liferay iOS SDK
+//
+//  Created by Victor Galan on 16/02/2018.
+//  Copyright © 2018 Liferay Inc. All rights reserved.
+//
+
+
+@class LRSession;
+
+@protocol LRCookieCallback <NSObject>
```

Comment:
Can you reuse LRCallback instead of creating a new one?

Reply:
LRCallback onSuccess method returns an id type because it could return any type, but LRCookieCallback is always returning a LRSession